src/database/db_get.js

```diff
@@ -0,0 +1,14 @@
+const connect = require('./db_connect');
+
+const get = {};
+
+get.memories = (userId, callback) => {
+  connect.query('SELECT memories.id, memories.heading, memories.likes, memories.avgRating, memories.visits, memories.tag, memories.memory FROM memories INNER JOIN users ON users.id = memories.user_id WHERE users.id = $1', [userId], (err, res) => {
```

Comment:
Can you pull this string out into a multiline variable and add indentation + new lines